pages/sponsor/prospectus.vue

```diff
@@ -131,65 +301,266 @@
                 </tr>
             </tbody>
         </table>
-
-        <!-- Special Sponsorship -->
-        <h2 id="special">
-            {{ $t('specialSponsorship-title') }}
-        </h2>
-        <div
-            v-for="(type, i) in specialSponsorshipTypes"
-            :key="`prospectus_special_${i}`"
-        >
-            <h3>
-                {{ $t(`specialSponsorship-${type}.0`) }}
-            </h3>
-            <ul class="list-disc">
-                <li
-                    v-for="(item, j) in $t(`specialSponsorship-${type}.1`)"
-                    :key="`prospectus_special_${i}_item_${j}`"
+        <core-hr></core-hr>
+        <!-- special sponsorship mobile -->
+        <p class="section-title text-sm md:text-lg lg:hidden">
+            {{ $t('specialSponsorship-title.0') }}
+        </p>
+        <div class="grid mt-8 lg:hidden">
+            <div
+                v-for="(specialSponsorShipTypes, i) in specialSponsorshipTypes"
+                :key="`prospectus_specials_${i}`"
+                class="grid sponsor-packages gap-x-3 mt-2"
+            >
+                <div class="section-title">
+                    <div class="mt-24 md:mt-36">
+                        <p class="text-xs md:text-sm">
+                            {{ $t('specialSponsorship-title.1') }}
+                        </p>
+                    </div>
+                    <p class="text-xs md:text-sm">
+                        {{ $t('specialSponsorship-title.2') }}
+                    </p>
+                    <p class="text-xs md:text-sm">
+                        {{ $t('specialSponsorship-title.3') }}
+                    </p>
+                    <p class="text-xs md:text-sm">
+                        {{ $t('specialSponsorship-title.4') }}
+                    </p>
+                </div>
+                <div
+                    v-for="specialSponsorShipType in specialSponsorShipTypes"
+                    :key="`prospectus_special_${specialSponsorShipType}`"
+                    class="text-xs md:text-sm"
                 >
-                    {{ item }}
-                </li>
-            </ul>
+                    <div
+                        class="w-16 md:w-28 h-16 md:h-28 bg-contain bg-no-repeat mb-8 mx-auto"
+                        :style="
+                            getImgStyle(
+                                specialSponsorshipTypeImgs[
+                                    `special-sponsorship-${specialSponsorShipType}`
+                                ],
+                            )
+                        "
+                    ></div>
+                    <p class="text-center">
```

Comment:
consider using for-loop through index `0` to `2` to reduce a few lines (and also pc view)